### ISSUE TITLE
Duplicate layer, contributors, sources on add

### ISSUE THREAD

**Issue by n0rdlicht** [cividi/spatial-data-package-export#52]
**Describe the bug**
When clicking the add layer/source/contributors buttons more than one (usually two, but sometimes more) new entries appear, requiring deletion of surplus entries.

**To Reproduce**
1. Setup a new snapshot, load one from the project file or disk
2. Add a layer or source or contributor

**Expected behavior**
One new entry.

**Screenshots**

<img width="470" alt="Screenshot 2021-07-08 at 16 55 41" src="https://user-images.githubusercontent.com/1425577/124944089-56c6d500-e00d-11eb-826f-a4e5ceb24d76.png">

**Desktop (please complete the following information):**
 - Plugin: 0.2.1 (beta release)
 - QGIS 3.18.3
 - Python: 3.8.7
 - OS: macOS 11.4

**Comment by n0rdlicht** [comment]
This seems to be not an issue when using QGIS 3.20.

**Comment by tjukanovt** [comment]
Yup. Seems to be working fine also on Windows 10 with version 3.18.2.
![contributor](https://user-images.githubusercontent.com/13899537/125242788-cf6faf00-e2f5-11eb-9bb5-e02c6064affb.gif)
